domains/drix10.is-cool.dev.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "Portfolio website",
+
+  "domain": "is-cool.dev",
+  "subdomain": "drix10",
+
+  "owner": {
+    "email": ""
+  },
+
+  "record": {
+    "CNAME": "f4d28d47-e408-4acf-a3d2-399ecd55a9bc.id.repl.co.",
+    "TXT": ["replit-verify=f4d28d47-e408-4acf-a3d2-399ecd55a9bc"]
```

Comment:
Sorry, you can't combine CNAME with any other records.

Reply:
ok, then the txt shoud be fine? do i need to remove the cname & keep only 1?

Reply:
You need to choose one.

Reply:
removed the txt record